maas/client/viscera/nodes.py

```diff
@@ -130,3 +137,14 @@ def as_region_controller(self):
                 'Cannot convert to `RegionController`, node_type is not a '
                 'region controller.')
         return self._origin.RegionController(self._data)
+
+    async def get_power_parameters(self):
+        """Get the power paramters for this node."""
+        data = await self._handler.power_parameters(system_id=self.system_id)
+        return data
+
+    async def set_power_parameters(self, power_type, power_parameters={}):
```

Comment:
I would rename this to `set_power` because its also setting the power_type not just the power parameters.

Update the docstring to:
`Set the power type and power parameters for this node."""

Also add typing:
```
async def set_power(self, power_type: str, power_parameters: Mapping[str, Any] = {})
```